lib/compact_index/versions_file.rb

```diff
@@ -27,20 +27,20 @@ def updated_at
     def create(gems)
       gems.sort!
 
-      File.open(@path, 'w') do |io|
+      File.open(@path, "w") do |io|
         io.write "created_at: #{Time.now.iso8601}\n---\n"
         io.write gem_lines(gems)
       end
     end
 
-private
+  private
 
     def gem_lines(gems)
       gems.each {|g| g.versions.sort! }
 
       gems.reduce("".dup) do |lines, gem|
         version_numbers = gem.versions.map(&:number_and_platform).join(",")
-        lines << gem.name <<
+        lines.dup << gem.name <<
```

Comment:
Sorry about this. I ended up introducing the dup when I was rebasing the commit. I have removed it now.